src/components/Terminal/Terminal.tsx

```diff
@@ -231,26 +235,38 @@ class Terminal extends React.Component<TerminalProps, {}> {
   }
 
   render() {
+    const userInfo = this.uiStore.getUserInfo();
+    const userName = userInfo ? userInfo.fullName : undefined;
+    const email = userInfo ? userInfo.email : undefined;
     return (
       <Shell>
-        <NotificationList />
-        {this.props.rootStore.modalStore.isModals ? (
-          <div>
-            <Backdrop />
-            <Modal modals={this.props.rootStore.modalStore.modals} />
-          </div>
-        ) : null}
-        {this.props.rootStore.sessionStore.sessionNotificationsBlockShown && (
-          <SessionNotificationComponent />
-        )}
-        <Header history={this.props.history} />
-        <Mosaic
-          renderTile={this.handleRenderTile}
-          onChange={this.handleChange}
-          resize={{minimumPaneSizePercentage: MIN_PANE_SIZE_PERCENTAGE}}
-          initialValue={this.state.initialValue}
+        <Header
+          // tslint:disable-next-line:jsx-no-lambda
+          onLogout={() => this.onLogout()}
```

Comment:
Why not `this.onLogout`?

Reply:
Because we are using this inside callback function